cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -289,11 +289,257 @@ func Test_TreePrefix(t *testing.T) {
 			addObjectRow("", tbl, tt.objectTree, tt.objectTree.GetRoot())
 			tbl.Render()
 
+			// Compare the output with the expected prefix.
+			// We only check whether the output starts with the expected prefix,
+			// meaning expectPrefix does not contain the full expected output.
 			g.Expect(output.String()).Should(MatchTable(tt.expectPrefix))
 		})
 	}
 }
 
+func Test_V1Beta2TreePrefix(t *testing.T) {
+	tests := []struct {
+		name         string
+		objectTree   *tree.ObjectTree
+		expectPrefix []string
+	}{
+		{
+			name: "Conditions should get the right prefix with multiline message",
+			objectTree: func() *tree.ObjectTree {
+				root := fakeObject("root",
+					withV1Beta2Condition(metav1.Condition{
+						Type:    "Available",
+						Status:  metav1.ConditionFalse,
+						Reason:  "NotAvailable",
+						Message: "first line must not be validated in the test\nsecond line",
+					}),
+				)
+				objectTree := tree.NewObjectTree(root, tree.ObjectTreeOptions{
+					V1Beta2: true,
+				})
+
+				o1 := fakeObject("child1",
+					withV1Beta2Condition(falseV1Beta2Condition("Available", "first line must not be validated in the test\nsecond line")),
+				)
+
+				o2 := fakeObject("child2",
+					withV1Beta2Condition(falseV1Beta2Condition("Available", "first line must not be validated in the test\nsecond line")),
+				)
+				objectTree.Add(root, o1)
+				objectTree.Add(root, o2)
+				return objectTree
+			}(),
+			expectPrefix: []string{
+				"Object/root              Available: False  NotAvailable",
+				"│                                                              second line",
+				"├─Object/child1          Available: False  NotAvailable",
+				"│                                                              second line",
+				"└─Object/child2          Available: False  NotAvailable",
```

Comment:
can we / should we check the second line for this object?

Reply:
We should, of course. Don't know why it's missing, seems like it got lost. Good catch 👍🏻